src/Microsoft.AspNet.Mvc.ViewFeatures/Logging/DefaultViewComponentInvokerLoggerExtensions.cs

```diff
@@ -40,10 +40,10 @@ public static void ViewComponentExecuting(this ILogger logger, ViewComponentCont
         public static void ViewComponentExecuted(
             this ILogger logger,
             ViewComponentContext context,
-            int startTime,
+            DateTime startTime,
             object result)
         {
-            var elapsed = new TimeSpan(Environment.TickCount - startTime);
+            var elapsed = DateTime.UtcNow - startTime;
```

Comment:
It's better than using `Environment.TickCounts` according to http://stackoverflow.com/a/8865560. I think the reason it doesn't use a `Stopwatch` is to avoid the allocation for logging.